### DISCUSSION TITLE
Errors when syncing yaap-twelve

### DISCUSSION THREAD

**Discussion by mhmorpheus** [ROM-builders/temporary#18855] [General]
hey,
https://cirrus-ci.com/task/6609299233832960
seems like the server has mistaken yaap for yaosp which is different from one and the other
the following is the error line:
error: Cannot checkout yaosp/vendor_google_gms: ManifestInvalidRevisionError: revision twelve in yaosp/vendor_google_gms not found'
hope you can fix it
thanks in advance

**Comment by Apon77** [comment]
issue from rom side,
